frontend/src/pages/faq.page.tsx

```diff
@@ -229,6 +229,58 @@ const Faq: NextPage = () => {
                   <p />
                 </Localized>
               </QAndA>
+              <QAndA
+                id="faq-acceptable-use"
+                question={l10n.getString(
+                  "faq-question-acceptable-use-question"
+                )}
+              >
+                <Localized
+                  id="faq-question-acceptable-use-answer-a-html"
+                  vars={{
+                    url: "https://www.mozilla.org/about/legal/acceptable-use/",
+                    attrs: "",
+                  }}
+                  elems={{
+                    a: (
+                      <a href="https://www.mozilla.org/about/legal/acceptable-use/" />
+                    ),
+                  }}
+                >
+                  <p />
+                </Localized>
```

Comment:
This is a small peculiarity of [`@fluent/react`](https://github.com/projectfluent/fluent.js/wiki/React-Bindings): it can only replace `<tags>` in translation strings via the `<Localized>` wrapper component, not via the `useLocalization` hook (that provides the `l10n` object).

The way that component works is: it takes its child element (`<p />`, in this case), and replaces its contents with the contents of the Fluent string (`faq-question-acceptable-use-answer-a-html`).

Also note that `vars` shouldn't actually be necessary: if the string just says `<my-tag>acceptable use policy</my-tag>`, then we could add `my-tag` to `elems` and set whatever attribute we need on the element there, just like we set the `href` attribute on the `<a>` element we replace the `<a>` tag with. However, because this string is also used on the current site, I had to use the `<a href="{ $url }" { $attrs }>` trick too.